mlir/include/mlir/Conversion/FuncToEmitC/FuncToEmitC.h

```diff
@@ -9,10 +9,12 @@
 #ifndef MLIR_CONVERSION_FUNCTOEMITC_FUNCTOEMITC_H
 #define MLIR_CONVERSION_FUNCTOEMITC_FUNCTOEMITC_H
 
+#include "mlir/Transforms/DialectConversion.h"
```

Comment:
I see other places just forward declare `class TypeConverter;` here instead of doing this include